storops/unity/resource/host.py

```diff
@@ -311,6 +326,40 @@ def delete_ip_port(self, address):
                      .format(address, self.name))
         return resp
 
+    def update_initiators(self, iqns=None, wwns=None):
+        """Primarily for puppet-unity use.
+
+        Update the iSCSI and FC initiators if needed.
+        """
+        # first update the iqns
+        iqns = iqns if iqns else []
```

Comment:
You could initialize it to `set` since you are converting them to set later anyway.